SPECS/sm.spec

```
@@ -63,7 +63,7 @@ BuildRequires: gcc
 Requires(post): systemd
 Requires(preun): systemd
 Requires(postun): systemd
-Requires: xenserver-multipath
+Requires(post): xenserver-multipath
```

```suggestion
Requires: xenserver-multipath
Requires(post): xenserver-multipath
```

I'm not sure `Requires(post)` implies `Requires`. IIRC, with `Requires(post)` alone, you can then uninstall the requirement as you didn't mention it is required for normal operation.